examples/renameimage.py

```diff
@@ -38,44 +97,100 @@
 
     glancesync = GlanceSync()
     glancesync.init_logs()
+    master = glancesync.master_region
+
+    master_images = glancesync.get_images_region(master)
+
+    old_image = None
+    new_image = None
+    for image in master_images:
+        if image.name == sys.argv[1]:
+            if old_image:
+                msg = 'There are more than an image with the old name in the' \
+                      ' master region'
+                logging.error(msg)
+                sys.exit(-1)
+            else:
+                old_image = image
+        elif image.name == sys.argv[2]:
+            if new_image:
+                msg = 'There are more than an image with the new name in the' \
+                      ' master region'
+                logging.error(msg)
+                sys.exit(-1)
+            else:
+                new_image = image
+
+    if old_image and new_image:
+        msg = 'Both an image with the new name and the old new exists in the' \
+            ' master region'
+        logging.error(msg)
+        sys.exit(-1)
+
+    if new_image:
+        master_image = new_image
+    elif old_image:
+        master_image = old_image
+        old_image.name = sys.argv[2]
```

Comment:
The confusion is probable because a bad chosen variable names. old_image should be image_with_old_name, new_image, image_with_new_name.

The idea is: search for images with old/new name. It both exist, this is an error. If only image with the new name exits, the image is already renamed, use this image as reference to use the checksum, is_property and other properties. If only image with the old name exists, the same, but first update the name (rename it)

I will change the names